### PR TITLE
switch Syntex error

### PR DESCRIPTION
**Pull request by LeeYuRi1**
switch Syntex error(tab)

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [ef6a2ae6cf32195153a4f4e9bd1d65f66a171c4e] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [ef6a2ae6cf32195153a4f4e9bd1d65f66a171c4e] - Code Rule Check (C++) OK.

File: external/json/cJSON.c [hunk 1]
```diff
@@ -721,43 +721,42 @@ static cJSON_bool parse_string(cJSON * const item, parse_buffer * const input_bu
...
-                    *output_pointer++ = '\t';
-                    break;
-                case '\"':
-                case '\\':
-                case '/':
-                    *output_pointer++ = input_pointer[1];
-                    break;
+            switch (input_pointer[1]) {
+            case 'b':
+            	*output_pointer++ = '\b';
```
Comment (rzr) [inline external/json/cJSON.c]: make sure to preserve style,

cleanup change are preffered is separated changes

**Comment by sunghan-chang** [comment]
@LeeYuRi1 This fix is not approprietary. Let's close now and then when you are free, please fix again and reopen this. Thank you for contribution.

**Pull request closed without merging**